### PR TITLE
Added Dribbblish dynamic theme

### PR DESCRIPTION
**Pull request by JulienMaille**
See readme for a gif and more details

### CODE REVIEW

**Comment by morpheusthewhite** [comment]
Am I missing something or there are no new information (from the original theme)?

**Comment by UrSok** [comment]
I think the only difference is the gif.

**Comment by UrSok** [comment]
@JulienMaille what is the percent btw?
image

**Comment by JulienMaille** [comment]
> Am I missing something or there are no new information (from the original theme)?

The difference with original theme is mostly that the accent color will follow the current - album cover (look at the gif in the readme) and the cover is visible in the background.

**Comment by JulienMaille** [comment]
> @JulienMaille what is the percent?
> image
> 

The percent is the popularity of the track given by spotify

**Comment by UrSok** [comment]
> > @JulienMaille what is the percent?
> > image
> 
> The percent is the popularity of the track given by spotify.
> Do you like my mod?

Yeah, using it, it looks very nice.

**Comment by UrSok** [comment]
@JulienMaille It looks like this cover is broke the team.
image

**Comment by JulienMaille** [comment]
Thanks for reporting, can you try my fix?

**Comment by UrSok** [comment]
It worked.

**Comment by UrSok** [comment]
@JulienMaille on launch the color is default. Can you make it to correspond to the cover?

**Comment by JulienMaille** [comment]
It should already work like that but something is wrong with events on startup.

File: Dribbblish-dynamic/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Dribbblish-dynamic
+
```
Comment (morpheusthewhite) [inline Dribbblish-dynamic/README.md:2]: Add explanation of main difference from Dribbblish

File: Dribbblish-dynamic/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## Screenshots
+#### Dark
+![demo1](./color-match-bg.gif)
```
Comment (morpheusthewhite) [inline Dribbblish-dynamic/README.md:5]: Add preview for other color schemes as well

File: Dribbblish-dynamic/dribbblish.js [hunk 3]
```diff
@@ -0,0 +1,300 @@
+// Hide popover message
```
Comment (morpheusthewhite) [inline Dribbblish-dynamic/dribbblish.js:1]: Rename file to `dribblish-dynamic.js` to avoid ambiguity

File: Dribbblish-dynamic/README.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+|`sidebar_and_player_bg`| Background for the player|
+|`sidebar_indicator_and_hover_button_bg`| For the slider & selected items when you hover over it|
+|`cover_overlay_and_shadow`| Overlay for when you hover over the album covers|
+|`slider_bg`| The background for the slider|
+|`scrollbar_fg_and_selected_row_bg`| Color for the current selected row|
+|`active_control_fg`| Foreground for active control items|
+|`indicator_fg_and_button_bg`| Button text color|
+|`miscellaneous_bg`| The background color of toolips ("You're offline" etc)|
+|`miscellaneous_hover_bg`| Hover Color for the Tooltips|
+|`preserve_1`| Misc text colors|
```
Comment (morpheusthewhite) [inline Dribbblish-dynamic/README.md:84]: Say thanks to the author of the original theme (@khanhas )

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]
Also change the name of the theme to fit the convention of other themes (either to be Dribbblish-Dynamic or DribbblishDynamic

File: Dribbblish-dynamic/dribbblish.js (hunk 3 above)
Comment (morpheusthewhite) [reply Dribbblish-dynamic/dribbblish.js:1]: This means that you will need to update the instructions in the README too

File: Dribbblish-dynamic/README.md (hunk 1 above)
Comment (JulienMaille) [reply Dribbblish-dynamic/README.md]: Renamed to DribbblishDynamic and explained the main difference

File: Dribbblish-dynamic/dribbblish.js (hunk 3 above)
Comment (JulienMaille) [reply Dribbblish-dynamic/dribbblish.js]: renamed and updated README

File: Dribbblish-dynamic/README.md (hunk 4 above)
Comment (JulienMaille) [reply Dribbblish-dynamic/README.md]: Done. Didn't know khansas was the author, I thought it was you @morpheusthewhite !

File: Dribbblish-dynamic/README.md (hunk 2 above)
Comment (JulienMaille) [reply Dribbblish-dynamic/README.md]: Actually I never tried other schemes. I will test them and add previews

Comment (JulienMaille) [reply Dribbblish-dynamic/README.md]: done

**Review by morpheusthewhite — APPROVED** [review]

**Pull request merged**